azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/MessageView.kt

```diff
@@ -165,17 +166,50 @@ private fun BasicChatMessage(viewModel: MessageViewModel, dispatch: Dispatch) {
                     .align(alignment = Alignment.Bottom)
             ) {
                 // Display the Read Receipt
-                androidx.compose.animation.AnimatedVisibility(visible = viewModel.isRead) {
-                    Icon(
-                        painter =
-                        painterResource(
-                            id =
-                            R.drawable.azure_communication_ui_chat_ic_fluent_message_read_10_filled
-                        ),
-                        contentDescription = "Message Read",
-                        tint = ChatCompositeTheme.colors.unreadMessageIndicatorBackground,
-                        modifier = Modifier.padding(start = 4.dp)
-                    )
+                androidx.compose.animation.AnimatedVisibility(visible = viewModel.showStatusIcon) {
+
+                    when (viewModel.messageStatus) {
+                        MessageSendStatus.SENDING -> {
+                            Icon(
+                                painter =
+                                painterResource(
+                                    id =
+                                    R.drawable.azure_communication_ui_chat_ic_fluent_message_sending_10_filled
+                                ),
+                                contentDescription = "Message Failed To Send",
```

Comment:
Leave ContentDescription null here (explicitly).

we can notify Failed to send in the message semantic grouping, or by announcement. These content descriptions would interrupt flow of screen reader.